src/action.js

```diff
@@ -56,13 +58,13 @@ async function commit(params) {
   }
 
   const prefix = !!destinationFolder ? `${destinationFolder}/` : '';
-  const path = `${prefix}problems/${name}/solution.${LANG_TO_EXTENSION[submission.lang]}`
+  const path = `${prefix}problems/${name}/solution.md` // Markdown file for the problem with the solution at the end
```

Comment:
This is great, but I think some users might solve the problem in multiple languages - do you think it would make sense to just have the question data in its own .md file and then keep the solutions as separate files? 

One additional benefit is that one could easily see how many lines of code they have written in particular languages with GitHub stats if they're in the proper extension.

Reply:
`Here's the updated code that creates a separate 'question.md' file for the question data:`

![Question File](https://github.com/joshcai/leetcode-sync/assets/36575352/edfa9c42-2ac8-47f8-863b-aa036501e5d9)

`Additionally, we now have individual solution files with the language as their extension:`

![Solution File](https://github.com/joshcai/leetcode-sync/assets/36575352/e2937dd6-b1b3-48d6-ac84-5e4437b67886)

Feel free to share any other suggestions you might have!

Reply:
Output looks great!